CHANGES

```diff
@@ -1,4 +1,4 @@
-2.0~beta
+2.0~alpha4
```

Comment:
oh, sorry; you managed to get to beta already ? (that was merged earlier today)
Just edit the field in `~/.opam/config` to put `alpha4` instead and you should be fine.

Thanks again for trying the bleeding edge!